benchmark/child_process/child-process-read.js

```diff
@@ -19,7 +19,7 @@ function main(conf) {
   const dur = +conf.dur;
   const len = +conf.len;
 
-  const msg = '"' + Array(len).join('.') + '"';
+  const msg = `"${'.'.repeat(len)}"`;
```

Comment:
This is doing something different from before, it should be `'.'.repeat(len - 1)`.